gateway-server/src/test/java/org/apache/knox/gateway/services/factory/TokenStateServiceFactoryTest.java

```diff
@@ -43,22 +44,31 @@ public void testBasics() throws Exception {
 
   @Test
   public void shouldReturnDefaultAliasService() throws Exception {
```

Comment:
Method name indicates an alias service test, but the implementation is a TokenStateService test.